SQL.txt

```diff
@@ -0,0 +1,2686 @@
+1) List all the columns of the Salespeople table. 
+Ans- 
+show columns 
+from 
+  salespeople;
+
++-------+--------------+------+-----+---------+-------+
+| Field | Type         | Null | Key | Default | Extra |
++-------+--------------+------+-----+---------+-------+
+| snum  | int          | YES  |     | NULL    |       |
+| sname | varchar(100) | YES  |     | NULL    |       |
+| city  | varchar(100) | YES  |     | NULL    |       |
+| comm  | int          | YES  |     | NULL    |       |
++-------+--------------+------+-----+---------+-------+
+
+desc salespeople;
++-------+--------------+------+-----+---------+-------+
+| Field | Type         | Null | Key | Default | Extra |
++-------+--------------+------+-----+---------+-------+
+| snum  | int          | YES  |     | NULL    |       |
+| sname | varchar(100) | YES  |     | NULL    |       |
+| city  | varchar(100) | YES  |     | NULL    |       |
+| comm  | int          | YES  |     | NULL    |       |
++-------+--------------+------+-----+---------+-------+
+
+2) List all customers with a rating of 100. 
```

Comment:
I think the data inserted was wrong.
The cname here should be Clemens. Please check.